src/test/resources/regressions/features/interfaces/proofGenForImport.gobra

```diff
@@ -11,6 +11,9 @@ package client
 import pkg "proofGenForImportPkg"
 
 func client() {
-    impl := &pkg.ImplPkgState{}
```

Comment:
is it not guaranteed that `&pkg.ImplPkgState{}` is non-nil?

Reply:
This is probably hard to see in the diff: ImplPkgState is a struct that contains the interface, so `ImplPkgState{}` initializes the interface fields with the default value, meaning that the interface field is initialized with nil.